api/src/models/service_config/mod.rs

```diff
@@ -26,23 +26,26 @@
 use crate::models::service::ContainerType;
 use crate::models::Image;
 pub use environment::{Environment, EnvironmentVariable};
+use secstr::SecUtf8;
 use serde::Deserialize;
 use serde_value::Value;
 use std::collections::BTreeMap;
 use std::hash::Hash;
 use std::path::PathBuf;
-
```

Comment:
Please, keep that line.

Reply:
White space maintained for better readability.